packages/lisk-transactions/src/transaction_types.ts

```diff
@@ -38,6 +38,7 @@ export interface TransactionJSON {
 	readonly asset: object;
 	readonly fee: string | number;
 	readonly id?: string;
+	readonly blockId?: string;
```

Comment:
shouldn't this be `readonly blockId?: string | null` ?

Reply:
should it accept `null`...? i think it's either exists or not